### PR TITLE
initial implementation for firewalld role

### PR DESCRIPTION
**Pull request by robled**

### CODE REVIEW

**Review by greg-hellings — APPROVED** [review]

File: tasks/main.yml [hunk 1]
```diff
@@ -1 +1,21 @@
-# role tasks
\ No newline at end of file
+# role tasks
+
+- name: open ports for zone "{{ zone }}" (IPv4+IPv6)
```
Comment (greg-hellings) [inline tasks/main.yml]: Is `"{{ zone }}"` the proper thing to have here?

File: tasks/main.yml [hunk 2]
```diff
@@ -1 +1,21 @@
...
+  firewalld:
+    port: "{{ item.port }}/{{ item.proto }}"
+    state: enabled
+    permanent: true
+    immediate: true
+    zone: "{{ firewalld_zone }}"
+  loop: "{{ firewalld_ports_open }}"
+
+- name: locally forward ports for zone "{{ zone }}" (IPv4+IPv6)
```
Comment (greg-hellings) [inline tasks/main.yml]: Ditto line 3.

File: tasks/main.yml (hunk 1 above)
Comment (robled) [reply tasks/main.yml]: It's not.  Will fix.

File: tasks/main.yml (hunk 2 above)
Comment (robled) [reply tasks/main.yml]: It's not.  Will fix.

**Review by greg-hellings — APPROVED** [review]

**Pull request merged**